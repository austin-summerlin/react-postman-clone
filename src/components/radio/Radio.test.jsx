import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Radio from '../radio/Radio';

describe('Radio Component', () => {
  afterEach(() => cleanup());
  it.only('should render Radio', () => {
    const { asFragment } = render(<Radio />);
    expect(asFragment()).toMatchSnapshot();
  });
});
