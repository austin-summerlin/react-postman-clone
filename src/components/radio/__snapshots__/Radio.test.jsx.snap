// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Radio Component should render Radio 1`] = `
<DocumentFragment>
  <div>
    <input
      id="get"
      name="method"
      type="radio"
      value="GET"
    />
    <label
      for="get"
    >
      GET
    </label>
    <input
      id="post"
      name="method"
      type="radio"
      value="POST"
    />
    <label
      for="post"
    >
      POST
    </label>
    <input
      id="put"
      name="method"
      type="radio"
      value="PUT"
    />
    <label
      for="put"
    >
      PUT
    </label>
    <input
      id="delete"
      name="method"
      type="radio"
      value="DELETE"
    />
    <label
      for="delete"
    >
      DELETE
    </label>
  </div>
</DocumentFragment>
`;
